lib/bowling.rb

```diff
@@ -0,0 +1,82 @@
+class Bowling 
+
+  def initialize
+    @frame = ""
+    @roll = ""
+    @pins = ""
+    @pins_array = []
+    @current_frame = []
+    @score_array = []
+    @spare = []
+    @strike = []
+  end 
+
+  # method with users input 
+  # def frames
+  #   puts "What frame are you currently playing?"
+  #   frame = gets.chomp.to_s
+  #   @frame = frame
+  #   binding.irb
+  # end
+
+  def frames(frame)
+    @frame = frame
+  end
+
+  def rolls(roll, pins)
+    @roll = roll
+    @pins = pins 
+    @pins_array << pins.to_i
+    @current_frame = @pins_array.last(2)
+  end 
+
+  def spare
+    @spare << 1 
+    return "Well Done! You scored a spare!" if @current_frame.sum == 10
+  end 
+
+  def strike
```

Comment:
Good idea definining your strikes and spares in separate methods. Perhaps they could be private?